### PR TITLE
build(deps): bump fastly-native-promises

### PR DESCRIPTION
**Pull request by trieloff**
See #755

### CODE REVIEW

**Comment by trieloff** [comment]
@stefan-guggisberg https://github.com/adobe/helix-publish/pull/760/commits/acc61bd33e0ffbd0e3baec50531d737590bbf3cf is the first change that got us over the deploy timeout issue. I think the diff tests failing in https://app.circleci.com/pipelines/github/adobe/helix-pages/3374/workflows/1dfc3c2e-7a2e-4f73-b34c-8a7986f5d810 might be a false positive, so I'm restarting.

**Comment by trieloff** [comment]
@tripodsan the diff test failure shows that https://spark-website--adobe.hlx.page/make/add-music-to-video is on the `breaking` branch, but https://spark-website--adobe.hlx-6.page/make/add-music-to-video is not. Is that intentional?

**Comment by trieloff** [comment]
Looks like it is intentional: https://github.com/adobe/spark-website/blob/main/helix-version.txt

But why doesn't the new version pick up the hint?

**Comment by trieloff** [comment]
Old trace: `vcl_recv; hlx_recv_init; hlx_strain(resolved); hlx_version_lock; hlx_block_recv; hlx_deny; hlx_allow; hlx_determine_request_type(existing:Preflight); hlx_type_preflight; hlx_owner; hlx_repo; hlx_ref; vcl_hash(spark-website--adobe.hlx.page/make/add-music-to-video); vcl_miss(spark-website--adobe.hlx.page/make/add-music-to-video); hlx_bereq; vcl_fetch(200: /make/add-music-to-video Preflight); hlx_headers_fetch; hlx_fetch_preflight; hlx_fetch_errors(200); hlx_fetch_static(none); hlx_fetch_error(200); vcl_deliver; hlx_headers_deliver; hlx_deliver_errors(200); hlx_deliver_type(Preflight); hlx_deliver_preflight(ok)(site-version=(null)/(null)); RESTART; vcl_recv; hlx_recv_init; hlx_strain(resolved); hlx_version_lock; hlx_block_recv; hlx_deny; hlx_allow; hlx_determine_request_type(none); hlx_type_dispatch; hlx_owner; hlx_repo; hlx_ref; hlx_root_path; hlx_index; hlx_github_static_owner; hlx_github_static_repo; hlx_github_static_ref; hlx_github_static_root(/htdocs); hlx_action_root; vcl_hash(spark-website--adobe.hlx.page/make/add-music-to-video); vcl_miss(spark-website--adobe.hlx.page/make/add-music-to-video); hlx_bereq; vcl_fetch(200: /make/add-music-to-video Dispatch); hlx_headers_fetch; hlx_fetch_errors(200); hlx_fetch_static(none); hlx_fetch_error(200); vcl_deliver; hlx_headers_deliver; hlx_deliver_errors(200); hlx_deliver_type(Dispatch); hlx_deliver_static(ok); hlx_deliver_redirect`

New trace: `vcl_recv; hlx_recv_init; hlx_strain(resolved); hlx_version_lock; hlx_block_recv; hlx_deny; hlx_allow; hlx_determine_request_type(existing:Preflight); hlx_type_preflight; hlx_owner; hlx_repo; hlx_ref; vcl_hash(spark-website--adobe.hlx-6.page/make/add-music-to-video); vcl_miss(spark-website--adobe.hlx-6.page/make/add-music-to-video); hlx_bereq; vcl_fetch(200: /make/add-music-to-video Preflight); hlx_headers_fetch; hlx_fetch_preflight; hlx_fetch_errors(200); hlx_fetch_static(none); hlx_fetch_error(200); vcl_deliver; hlx_headers_deliver; hlx_deliver_errors(200); hlx_deliver_type(Preflight); hlx_deliver_preflight(ok)(site-version=(null)/(null)); RESTART; vcl_recv; hlx_recv_init; hlx_strain(resolved); hlx_version_lock; hlx_block_recv; hlx_deny; hlx_allow; hlx_determine_request_type(none); hlx_type_dispatch; hlx_owner; hlx_repo; hlx_ref; hlx_root_path; hlx_index; hlx_github_static_owner; hlx_github_static_repo; hlx_github_static_ref; hlx_github_static_root(/htdocs); hlx_action_root; vcl_hash(spark-website--adobe.hlx-6.page/make/add-music-to-video); vcl_hit(spark-website--adobe.hlx-6.page/make/add-music-to-video); vcl_deliver; hlx_headers_deliver; hlx_deliver_errors(200); hlx_deliver_type(Dispatch); hlx_deliver_static(ok); hlx_deliver_redirect`

**Comment by trieloff** [comment]
New trace including https://github.com/adobe/helix-publish/pull/760/commits/6cde73e63a5f93fadd93e18da7d9b0331e9c1086

`X-Trace: vcl_recv; hlx_recv_init; hlx_strain(resolved); hlx_version_lock; hlx_block_recv; hlx_deny; hlx_allow; hlx_determine_request_type(existing:Preflight); hlx_type_preflight; hlx_owner; hlx_repo; hlx_ref; vcl_hash(spark-website--adobe.hlx-4.page/make/add-music-to-video); vcl_miss(spark-website--adobe.hlx-4.page/make/add-music-to-video); hlx_bereq; vcl_fetch(200: /make/add-music-to-video Preflight); hlx_headers_fetch; hlx_fetch_preflight; hlx_fetch_errors(200); hlx_fetch_static(none); hlx_fetch_error(200); vcl_deliver; hlx_headers_deliver; hlx_deliver_errors(200); hlx_deliver_type(Preflight); hlx_deliver_preflight(ok)(x-pages-version=breaking-202103/breaking-202103); RESTART; vcl_recv; hlx_recv_init; hlx_strain(resolved); hlx_version_lock; hlx_block_recv; hlx_deny; hlx_allow; hlx_determine_request_type(none); hlx_type_dispatch; hlx_owner; hlx_repo; hlx_ref; hlx_root_path; hlx_index; hlx_github_static_owner; hlx_github_static_repo; hlx_github_static_ref; hlx_github_static_root(/htdocs); hlx_action_root; vcl_hash(spark-website--adobe.hlx-4.page/make/add-music-to-video); vcl_hit(spark-website--adobe.hlx-4.page/make/add-music-to-video); vcl_deliver; hlx_headers_deliver; hlx_deliver_errors(200); hlx_deliver_type(Dispatch); hlx_deliver_static(ok); hlx_deliver_redirect`

The version is being loaded correctly: `x-pages-version=breaking-202103/breaking-202103); RESTART;`

File: src/publish.js [hunk 1]
```diff
@@ -9,6 +9,9 @@
  * OF ANY KIND, either express or implied. See the License for the specific language
  * governing permissions and limitations under the License.
  */
+// use HTTP1 if we are on Adobe I/O Runtime
+// eslint-disable-next-line no-underscore-dangle
+process.env.HELIX_FETCH_FORCE_HTTP1 = process.env.__OW_ACTIVATION_ID;
```
Comment (stefan-guggisberg) [inline src/publish.js:14]: I don't think this is necessary.

**Review by stefan-guggisberg — APPROVED** [review]

**Pull request merged**